lte/gateway/c/core/oai/common/redis_utils/CMakeLists.txt

```diff
@@ -12,7 +12,21 @@
 cmake_minimum_required(VERSION 3.7.2)
 
 add_library(redis_utils redis_client.cpp)
-target_link_libraries(redis_utils MAGMA_CONFIG COMMON cpp_redis tacopie protobuf)
+
+file(GLOB ABSL_LIBS "/usr/local/lib/libabsl*.a")
+
+target_link_libraries(redis_utils MAGMA_CONFIG COMMON cpp_redis tacopie 
+    ssl crypto z gpr
```

Comment:
This is inconsistent with the listing above. If those imports are the same everywhere, they should also be consistently formatted to ease understanding and review.